src/client/pages/ToeristischeVerhuurDetail/VakantieVerhuur.tsx

```diff
@@ -6,41 +6,109 @@ import { defaultDateFormat } from '../../../universal/helpers/date';
 import InfoDetail, {
   InfoDetailGroup,
 } from '../../components/InfoDetail/InfoDetail';
+import { PageContent } from '../../components';
+import styles from './ToeristischeVerhuurDetail.module.scss';
+import StatusLine, {
+  StatusLineItem,
+} from '../../components/StatusLine/StatusLine';
+import { useMemo } from 'react';
+
+function useStatusLineItems(
+  vergunning?: ToeristischeVerhuur | ToeristischeVerhuurAfmelding
+) {
+  const statusLineItems: StatusLineItem[] = useMemo(() => {
+    if (!vergunning) {
+      return [];
+    }
+
+    const statusTrain = [
+      {
+        id: 'item-1',
+        status: 'Ontvangen',
+        datePublished: vergunning.dateRequest,
+        description: '',
+        documents: [],
+        isActive: false,
+        isChecked: true,
+      },
+      {
+        id: 'item-2',
+        status: 'Gemeld',
+        datePublished: vergunning.dateRequest,
+        description: '',
+        documents: [],
+        isActive: false,
```

Comment:
Deze kun je bepalen aan de hand van de status. `isActive = vergunning.status==='Gemeld'`

Reply:
Ik weet niet of er een decos status is die Gemeld als waarde heeft. Als ik het goed begrijp is dit hoe wij het noemen wanneer het in decos als 'afgehandeld' staat.

Reply:
we tonen namelijk ook altijd die twee bolletjes en de tweede is altijd actief en ze zijn allebei gechecked. Het is eigenlijk ook compleet betekenisloos